bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,39 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*): (?P<copyright>.*)UNKNOWN',
+        result_message='No license found.')
+class LicenseCheckBear:
+    """
+    Attempts to check the given file for a license, by searching the start
+    of the file for text belonging to various licenses. For Ubuntu/Debian
```

Comment:
Would layout the docs like this:
```
Attempts to check the given file for a license, by searching the start
of the file for text belonging to various licenses.

For Ubuntu/Debian ...
```
As the Ubuntu/Debian part is detailed :) Rule of thumb: The first line/sentence is the short description which shall describe the bear as short as possible, everything following an empty line is detailed description :)